src/glmtools.jl

```diff
@@ -514,7 +514,7 @@ function loglik_obs end
 loglik_obs(::Bernoulli, y, μ, wt, ϕ) = wt*logpdf(Bernoulli(μ), y)
 loglik_obs(::Binomial, y, μ, wt, ϕ) = logpdf(Binomial(Int(wt), μ), Int(y*wt))
 loglik_obs(::Gamma, y, μ, wt, ϕ) = wt*logpdf(Gamma(inv(ϕ), μ*ϕ), y)
-# In Distributions.jl, a Geometric distribution characterizes the number of failures before 
+# In Distributions.jl, a Geometric distribution characterizes the number of failures before
```

Comment:
Can you revert all these spacing changes? I think the ones affecting doctests below are the reason for failures.

Reply:
Ooof, yeah, sorry. I have my editor set up to trim trailing white space, which is generally great but can muck up docstring tests (and hit a huge number of unrelated lines in codebases).